memote/suite/cli/runner.py

```diff
@@ -83,7 +86,7 @@ def cli():
 @click.option("--filename", type=click.Path(exists=False, writable=True),
               default="result.json", show_default=True,
               help="Path for the collected results as JSON.")
-@click.option("--location", type=str, envvar="MEMOTE_LOCATION",
+@click.option("--location", envvar="MEMOTE_LOCATION",
```

Comment:
Do I interpret this right, that users will still have to provide to do `--location Results` when they run any history related CLI commands?

I think it would be nice if we could leave out `--location` as an additional parameter when the user invokes the CLI from inside a memote git repo, but I'm OK with this too.

Reply:
See above regarding configuration.